docs/examples.rst

```diff
@@ -1486,11 +1486,11 @@ Clean guests
 
 The subcommand ``clean guests`` aims to stop all running guests.
 By default, runs are taken from ``/var/tmp/tmt``, this can be
-changed using an argument to the subcommand:
+changed using ``--workdir-root`` to the subcommand:
```

Comment:
The sentence no longer sounds plausible, how about "changed using \`\`--workdir-root\`\` option of the subcommand:"?